linters/stylelint/plugin.yaml

```diff
@@ -1,5 +1,15 @@
 version: 0.1
 lint:
+  # When enabling stylelint, you should also add the stylelint plugin configs,
+  # like so:
+  #
+  #   lint:
+  #   enabled:
+  #     - stylelint@15.3.0:
+  #         packages:
+  #           - stylelint-config-standard@31.0.0
+  #           - stylelint-config-standard-scss@7.0.1
+  #
   definitions:
```

Comment:
Because you've added a direct config, you must set `is_recommended: false` (this is causing the repo_tests failure)